### DISCUSSION TITLE
AI創意機器人競賽

### DISCUSSION THREAD

**Discussion by number32984** [1101-GS3073-C/Python-Community-GS3073-C#623] [General]
我們小學的時候還在玩溜滑梯，他們國小就在學ai了，真的是時代的差距阿!
https://tw.news.yahoo.com/%E5%98%89%E7%BE%A9%E7%B8%A3%E5%BA%9C%E8%BE%A6%E7%90%86%E9%A6%96%E5%B1%86a-i%E5%89%B5%E6%84%8F%E6%A9%9F%E5%99%A8%E4%BA%BA%E7%AB%B6%E8%B3%BD%EF%BC%8Cscratch%E7%A8%8B%E5%BC%8F%E6%95%99%E8%82%B2%E8%90%BD%E5%9C%B0%E6%89%8E%E6%A0%B9-134659901.html

**Comment by magicmicheal** [comment]
嘉義首屆AI創意機器人競賽，scratch程式教育落地扎根

**Comment by awehao** [comment]
我小學的時候沒有玩溜滑梯歐，不過也沒有在做這麼專業的事，切身的感受到時代在進步，不過這也是少數人拉，希望他們都能從中感到快樂，而不是被老師父母逼迫，小學生就是要快樂的阿，不然上了國中就知道台灣的教育體制如何了

**Comment by larry40305** [comment]
亞洲家長：叫你學你就學

**Comment by Yilin20021016** [comment]
但更多的家長會叫你學一堆沒什麼用的東西

**Comment by Bonus502530** [comment]
雖然我是資電院的學生，不過其實在大學以前的求學階段很少接觸這方面的知識，頂多只有高中電腦課有教一些些C++基礎語法和暑假學校辦的資安營，但是我的同學雖然有有些人跟我一樣大學前不曾接觸過相關的知識，不過大多數的同學大部分在高中都有完整學過。我覺得這跟學校擁有的資源有蠻大的關係，因為我高中時是就讀私校，學校只著重在課業上很少有機會能接觸到課程以外的知識。

**Comment by bww0401** [comment]
怎麼感覺現在的小孩越來越聰明了...
